### PR TITLE
archetype-react-app: [minor][feat]Chrome Headless

### PR DESCRIPTION
**Pull request by didi0613**
deprecate PhantomJS, use Chrome headless.
Use phantomJS as default for users and adding chrome headless as an option.

If the user specify the `ENABLE_CHROME_HEADLESS=true`, will use chrome headless for karma runner. Otherwise loads phantomJS as default.

### CODE REVIEW

**Comment by donsalari** [comment]
👍

**Comment by jchip** [comment]
please rebase to latest

File: samples/universal-material-ui/package.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
     "react-tap-event-plugin": "^2.0.0"
   },
   "devDependencies": {
-    "electrode-archetype-react-app-dev": "^2.3.0",
+    "electrode-archetype-react-app-dev": "../../packages/electrode-archetype-react-app-dev",
```
Comment (jchip) [inline samples/universal-material-ui/package.json]: please revert this file

File: samples/universal-react-node/package.json [hunk 2]
```diff
@@ -46,7 +46,7 @@
     "uuid": "^3.0.1"
   },
   "devDependencies": {
-    "electrode-archetype-react-app-dev": "^2.0.1",
+    "electrode-archetype-react-app-dev": "../../packages/electrode-archetype-react-app-dev",
```
Comment (jchip) [inline samples/universal-react-node/package.json]: please revert this file

**Pull request merged**